examples/preprocess_example_module/preprocess_example_module.info.yml

```diff
@@ -2,4 +2,5 @@ name: Preprocess example module
 type: module
 description: Registers an extra preprocess event.
 core: 8.x
```

Comment:
This one can go right?

Reply:
Probably, though I would make sure that the minimum core version [is 8.7.7 for when they added ```core_version_requirement```](https://www.drupal.org/node/3070687), just so you don't end up with the one person who still hasn't updated Drupal opening an issue about it failing. :P

Reply:
I removed all the other `core` lines from the info files already 😉. This one is not a real module so basically forgot 😅.

Reply:
Fair enough!